### PR TITLE
Feature/heal submission from start date

### PR DESCRIPTION
**Pull request by cecemel**
In some cases; we'd like to only be able to heal a subset of the submissions. This PR allows the ability to start healing for submission where sentDate >= then a provided date.

To see in in action:
```
submissions-dispatcher:
  image: lblod/worship-submissions-graph-dispatcher-service:0.13.0.rc-sentDate-1
```
Once up and running:
```
drc exec submissions-dispatcher wget http://localhost/heal-submission?sentDateSince=2023-12-13
```

### CODE REVIEW

**Review by Poltergeistz — APPROVED** [review]
Works like expected with date samples (tested with date from today and one week ago)

File: README.md [hunk 1]
```diff
@@ -50,10 +50,21 @@ ENABLE_HEALING : enables healing, defaults to false
...
 ### POST /delta
 Triggers the preparation of a submission for the export when a resource is sent.
-### [DEBUG] GET /manual-dispatch?subject=http://bar
-Triggers a manual dispatch.
-Meant for debugging only.
-If no parameters are provided, will dispatch all submissions in DISPATCH_SOURCE_GRAPH.
-### [DEBUG] GET /heal-submission?subject=http://bar
-Triggers the healing of one submission, or all of them if no param is given
-Meant for debugging only.
+#### DEUBG API
```
Comment (Riadabd) [inline README.md]: Small typo
```suggestion
#### DEBUG API
```

File: app.js [hunk 2]
```diff
@@ -98,28 +105,59 @@ app.get("/manual-dispatch", async function (req, res) {
...
     return res.status(201).send();
   }
+
+  let submissions;
+
+  if(req.query.sentDateSince) {
+    let sentDateSince = new Date(req.query.sentDateSince).toISOString().split('T')[0];
+    console.log(`Received "?sentDateSince" paramater: ${req.query.sentDateSince}`);
+    console.log(`Converted to short date: ${sentDateSince}`);
+    console.log(`Healing will be applied submissions with sent date >= ${sentDateSince}`);
```
Comment (Riadabd) [inline app.js]: Small typo

```suggestion
    console.log(`Healing will be applied to submissions with sent date >= ${sentDateSince}`);
```

**Review by Riadabd — APPROVED** [review]
There is a small typo in the one of the `console.log` statements, but everything looks great! 

If I run the healing beyond a specific date, nothing gets healed (as expected). And if I run healing starting from a specific date, only the relevant submissions on and past that date get healed.

**Pull request merged**